lib/teamsnap.rb

```diff
@@ -1,3 +1,10 @@
+require "teamsnap/version"
+require "teamsnap/error"
+require "teamsnap/configuration"
+require "teamsnap/client"
+require "teamsnap/collection"
+require "teamsnap/item"
+
```

Comment:
Ah, gotcha, was pointing it in the direction of other popular api wrappers, and didn't know the background. Can collapse down again if it doesn't get too big.